CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,15 @@
+# Hibernate ORM Code of Conduct
+
+All participants in Hibernate projects are expected to adhere to the Commonhaus Foundation Code of Conduct. Please ensure you are familiar with its guidelines and expectations, as it’s essential for maintaining a positive and collaborative environment.
+The complete Code of Conduct, can be reviewed [here](https://www.commonhaus.org/policies/code-of-conduct/).
+
+## Hibernate ORM Contact information
+
+- To report an issue, CoC reports should be sent to (**TODO**) `report@PROJECT-mailing-list`. We will follow [CF CoC procedures][coc-reports] when handling CoC reports.
```

Comment:
`hibernate-dev@lists.jboss.org` if this really needs to be a mailing list.

`admin@hibernate.org` otherwise.

Reply:
I'm not sure if perhaps we should have an organization-wide new mailing list for this that conforms to the template format of 'report@url' ?

Reply:
Perhaps. But right now I don't think we can (easily) set up new mailing lists on https://lists.jboss.org , and in any case such mailing list would probably be public, which isn't ideal for reports/whistleblowing.

I think admin@hibernate.org will do for now, and let's create a task to move it to a new mailing list after we've moved the mailing lists to Commonhaus infra? I didn't even look into that yet.